virtcontainers/pkg/cgroups/manager.go

```diff
@@ -331,3 +331,24 @@ func (m *Manager) RemoveDevice(device string) error {
 	m.Unlock()
 	return fmt.Errorf("device %v not found in the cgroup", device)
 }
+
+func (m *Manager) UpdateCpuSets(cpuset string) error {
+	cgroups, err := m.GetCgroups()
+	if err != nil {
+		return fmt.Errorf("could not get original cgroups: err: %v", err)
+	}
+
+	m.logger().WithField("map-length", len(m.GetPaths())).Debug("EGERNST")
+
+	for key, cgroupPath := range m.GetPaths() {
+		m.logger().WithFields(logrus.Fields{
+			"key":  key,
+			"path": cgroupPath,
+		}).Debug("EGERNST")
+	}
+	m.Lock()
+	cgroups.CpusetCpus = cpuset
```

Comment:
No validation on `cpuset`? Can it be blank? Can we check the expected format, etc?

Reply:
An empty string is normal/expected.  Validation is done @ the sandbox via CPUSets package, and I expect we'll just have an error, if the format somehow was broken, when we try to 'apply' the cgroup (see err handling there).